web/src/cypress/support/rotation.ts

```diff
@@ -48,69 +49,60 @@ type Part = { pos: number }
 const sortParts = (a: Part, b: Part) => (a.pos < b.pos ? -1 : 1)
 
 function createRotation(rot?: RotationOptions): Cypress.Chainable<Rotation> {
-  const query = `mutation createRotation($input: CreateAllInput!){
-          createAll(input: $input) {
-            rotations {
+  const query = `mutation createRotation($input: CreateRotationInput!){
+          createRotation(input: $input) {
+            
               id
               name
               description
-              timeZone: time_zone
-              shiftLength: shift_length
+              timeZone
+              shiftLength
               type
               start
-              users: participants {
-                pos: position
-                user {
-                  id
-                  name
-                  email
-                }
+              isFavorite
+              users {
+                id
+                name
+                email
               }
-            }
+            
           }
       }`
 
   return cy.fixture('users').then(users => {
     if (!rot) rot = {}
-    const ids = c.pickset(users, rot.count).map((usr: any) => usr.id)
-    const parts = ids.map(id => ({ rotation_id: 'rot', user_id: id }))
+    const ids = users.map((user: any) => user.id)
```

Comment:
This should respect the `count` field of `RotationOptions`. Use `c.pickset` to select a random sample of the provided number of users.